MediaBrowser.XbmcMetadata/Parsers/MovieNfoParser.cs

```diff
@@ -122,18 +122,9 @@ private void ParseSetXml(string xml, Movie movie)
             using (var stringReader = new StringReader("<set>" + xml + "</set>"))
             {
                 // These are not going to be valid xml so no sense in causing the provider to fail and spamming the log with exceptions
-                var settings = new XmlReaderSettings()
-                {
-                    ValidationType = ValidationType.None,
-                    CheckCharacters = false,
-                    IgnoreProcessingInstructions = true,
-                    IgnoreComments = true
-                };
-
                 try
                 {
-                    // Use XmlReader for best performance
-                    using (var reader = XmlReader.Create(stringReader, settings))
+                    using (var reader = XmlReader.Create(stringReader, GetXmlReaderSettings()))
```

Comment:
Why is `GetXmlReaderSettings()` used in only one or two places? I think it should be used everywhere settings are created...

Reply:
You're still making that settings object everywhere else... maybe fix it for now anyway?